### ISSUE TITLE
Show AS Names in MAT

### ISSUE THREAD

**Issue by fortuna** [ooni/explorer#967]
Please add the names of the ASes to the MAT.

When I [reported on the blocking of Youtube in Russia](https://x.com/vinifortuna/status/1824541362970300876), I had to do some manual work to add the AS names, which was time consuming:

![image](https://github.com/user-attachments/assets/8b879c60-717e-46b9-a27e-68f4c01efea8)

It would be much easier on the user if OONI added the names instead. I didn't have the same patience to annotate my reports on the blocking of X in Brazil.

This is particularly problematic when a Country has tons of ASes and you want to find the more important ones quickly.

**Comment by hellais** [comment]
Yeah this would be a great thing to have, unfortunately ATM we don't store the AS name inside of the table that's being queried to produce the MAT chart. This is however being addressed by the new data pipeline, so once we have that going we should be able to use it to do that.
